install

```diff
@@ -137,6 +137,15 @@ def git
   @git
 end
 
+def user_only_chmod?(d)
+  return false unless File.directory?(d)
+  mode = File.stat(d).mode & 0b111_111_111
```

Comment:
Hex wouldn't be readable:
mode = File.stat(d).mode & 0x1ff
Octal is fine:
mode = File.stat(d).mode & 0777
I prefer binary, because it visually represents every bit flag (rwx) for every group (ugo).
Do you see huge difference between octal and binary?